### PR TITLE
Dynamic LR change

### PR DESCRIPTION
**Pull request by amogh7joshi**
Dynamically update the learning rate

### CODE REVIEW

File: yolo_uda/training/models.py [hunk 1]
```diff
@@ -62,9 +62,27 @@ def create_modules(module_defs: List[dict]) -> Tuple[dict, nn.ModuleList]:
...
+    # manually select which steps to decay the LR at (and by what value)
+    if "steps" in hyperparams and "scales" in hyperparams:
+        hyperparams.update({
+            'lr_steps': list(zip(map(int,   hyperparams["steps"].split(",")),
+                                 map(float, hyperparams["scales"].split(","))))
+        })
+
+    # decay by the value `lr_gamma` every N steps or every N epochs
+    elif "lr_gamma" in hyperparams and "lr_step" in hyperparams:
+        hyperparams.update({
```
Comment (mihamerstan) [inline yolo_uda/training/models.py:76]: Are L76 and L81 doing anything?  Do we need to update those values with their existing values?

Comment (mihamerstan) [reply yolo_uda/training/models.py:76]: Ah never mind, I see now. They were strings before. Ignore this.
